components/ShoppingItem/ShoppingItem.js

```diff
@@ -3,49 +3,63 @@ import Link from "next/link";
 import { useState } from "react";
 
 export default function ShoppingItem({
-  id,
   onDeleteItem,
   shoppingItem,
   onEditItem,
   onChangeMode,
```

Comment:
See my comment below.

```suggestion
```

Reply:
see my earlier comment: I need it here to open the form